docs/learn/learn-async-backing.md

```diff
@@ -0,0 +1,138 @@
+---
+id: learn-async
+title: Asynchronous Backing
+sidebar_label: Asynchronous Backing
+description: A brief overview of asynchronous backing, and how it affects Polkadot's scalability.
+keywords: [parachains, slots, backing, parablock]
+slug: ../learn-async
+---
+
+Asynchronous backing is a mechanism that introduces a _pipeline_ for parachain **<->** relay chain
+communication. This pipeline will allow collators to include more data in parachain blocks while
+reducing latency (from 12 to 6 seconds for parablock validation).
+
+It has three overarching goals:
+
+1. Decrease parachain blocks (parablock) validation time to **6 seconds** from **12 seconds**
+2. Increase the amount of usable blockspace by a factor of 5-10, meaning more state changes are
+   allowed per relay chain block.
+3. Allow for parablocks to be re-proposed to the network if they should be included initially on the
```

Comment:
May be I am not understanding this correctly. Why should parablocks be re-proposed if they should have been included in the first attempt?

```suggestion
3. Allow for parablocks to be re-proposed to the network if they are not included initially on the
```